src/app/App.tsx

```diff
@@ -1,18 +1,22 @@
-import { useState } from 'react';
 import { Container } from '@mui/material';
 
 import { Header } from 'ui';
 import { usePageTitle } from 'hooks';
 import { AppRoutes } from 'routing/AppRoutes';
+import { useAuth } from 'hooks/useAuth/useAuth';
 
 export const App = () => {
-  const [isLoggedIn, setIsLoggedIn] = useState(false);
-
   usePageTitle();
+  const { isLoggedIn, setToken } = useAuth();
 
   return (
     <>
-      <Header isLoggedIn={isLoggedIn} />
+      <Header
+        isLoggedIn={isLoggedIn}
+        onLogoutClick={() => {
+          setToken();
```

Comment:
Mhm, rozumiem, że ma go zerować, tak? To może lepiej tutaj wrzucić explicit `setToken(null)`?